### DISCUSSION TITLE
How to Configure Ruler's rules via file in deployment time with helm charts

### DISCUSSION THREAD

**Discussion by miladjahandideh** [grafana/mimir#2825] [Help and support]
We are using your [helm charts](https://artifacthub.io/packages/helm/grafana/mimir-distributed) ``chart_version: '2.0.14'`` to deploy Mimir and also ``multitenancy_enabled: false``

Now we want to enable `Ruler` and to add `Recording Rules` but want to add rule files via our IaC, not Grafana dashboards. Is it possible to deploy rules when we are deploying Mimir via helm chart like Prometheus?

**Comment by dimitarvdimitrov** [comment]
Not at this stage. There is an open issue with two PRs as potential fixes. And we have an issue on this https://github.com/grafana/helm-charts/issues/1240
